api-tests/checkout_payment_wallet_api.cards.tests.json

```diff
@@ -0,0 +1,216 @@
+							"    pm.environment.set(\"CHECKOUT_NONCE\", nonce);",
+							"    pm.environment.set(\"CHECKOUT_REDIRECT_URL\", urlRedirect);",
+							"});"
+						],
+						"type": "text/javascript",
+						"packages": {}
+					}
+				}
+			],
+			"request": {
+				"auth": {
+					"type": "noauth"
+				},
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "{{ENV}}/checkout/auth-service/v1/auth/login?recaptcha={{FAKE_RECAPTCHA}}",
+					"host": [
+						"{{ENV}}"
+					],
+					"path": [
+						"checkout",
+						"auth-service",
+						"v1",
+						"auth",
+						"login"
+					],
+					"query": [
+						{
+							"key": "recaptcha",
+							"value": "{{FAKE_RECAPTCHA}}"
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Checkout OneIdentity Redirect",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"var htmlResponse = pm.response.text();",
+							"var codeMatch = htmlResponse.match(/code=([^&]+)/);",
+							"if (codeMatch) {",
+							"    var code = codeMatch[1];",
+							"    pm.environment.set(\"CHECKOUT_AUTH_CODE\", code);",
+							"    console.log(\"Auth code extracted: \" + code);",
+							"} else {",
+							"    pm.test(\"Code should be present in the HTML response\", function() {",
+							"        pm.expect(codeMatch).to.not.be.null;",
+							"    });",
+							"}"
+						],
+						"type": "text/javascript",
+						"packages": {}
+					}
+				}
+			],
+			"request": {
+				"auth": {
+					"type": "noauth"
+				},
+				"method": "GET",
+				"header": [],
+				"url": "{{CHECKOUT_REDIRECT_URL}}"
+			},
+			"response": []
+		},
+		{
+			"name": "Checkout Auth Token",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"[Checkout Auth] POST auth token 200 with auth code\", function () {",
+							"    pm.response.to.have.status(200);",
+							"    const responseJson = pm.response.json();",
+							"    const authToken = responseJson.authToken;",
+							"    pm.expect(authToken).to.be.a(\"string\");",
+							"    pm.environment.set(\"AUTH_TOKEN\", authToken);",
+							"});"
+						],
+						"type": "text/javascript",
+						"packages": {}
+					}
+				}
+			],
+			"request": {
+				"auth": {
+					"type": "noauth"
+				},
+				"method": "POST",
+				"header": [
+					{
+						"key": "Content-Type",
+						"value": "application/json"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "{\n    \"state\": \"{{CHECKOUT_STATE}}\",\n    \"authCode\": \"{{CHECKOUT_AUTH_CODE}}\"\n}",
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "{{ENV}}/checkout/auth-service/v1/auth/token",
+					"host": [
+						"{{ENV}}"
+					],
+					"path": [
+						"checkout",
+						"auth-service",
+						"v1",
+						"auth",
+						"token"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get wallets by user fiscal code",
```

Comment:
just leave this api in this collection to be run in UAT env. the above apis are specifically used by checkout-auth-service.

Here we can create a special auth token used just for api tests that will not be validated by checkout-auth-service b.e. (to be used only in UAT env)